terraform/cloudfoundry/elbs.tf

```diff
@@ -59,18 +59,18 @@ resource "aws_elb" "cf_uaa" {
   }
 
   health_check {
-    target              = "HTTPS:9443/healthz"
+    target              = "HTTP:82/"
     interval            = "${var.health_check_interval}"
     timeout             = "${var.health_check_timeout}"
     healthy_threshold   = "${var.health_check_healthy}"
     unhealthy_threshold = "${var.health_check_unhealthy}"
   }
 
   listener {
-    instance_port      = 9443
-    instance_protocol  = "https"
+    instance_port      = 443
+    instance_protocol  = "ssl"
     lb_port            = 443
-    lb_protocol        = "https"
+    lb_protocol        = "ssl"
     ssl_certificate_id = "${var.system_domain_cert_arn}"
```

Comment:
This is going to cause significant API downtime as the ELB will be reconfigured in the cf-terraform job, several minutes before the UAA nodes get redeployed. to register themselves in gorouter, and the router nodes get added to the ELB.

I think we need to rethink this to do a multi-pass deployment to avoid downtime.